src/main/java/de/zalando/aruha/nakadi/controller/EventPublishingController.java

```diff
@@ -75,6 +78,10 @@ private ResponseEntity postEventInternal(String eventTypeName, String eventsAsSt
         }
     }
 
+    private ThrowableProblem crateProblem(JSONException e) {
+        return Problem.valueOf(Response.Status.BAD_REQUEST, e.getMessage());
```

Comment:
For example there could be IOException https://github.com/stleary/JSON-java/blob/25a87975beb7e826b148b5f69d77337f68ae8832/JSONWriter.java#L121